src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js

```diff
@@ -53,6 +53,13 @@ describe('Autosuggest', () => {
             expect(toJson(wrapper)).toMatchSnapshot();
             expect(wrapper.find('ListItem')).toHaveLength(0);
         });
+        it('should initially render focused component with suggestions list correctly', () => {
+            suggestionsList = SUGGESTIONS.slice(1, 20);
+            wrapper.setProps({ suggestions: suggestionsList });
+            setFocusOnInput();
+            expect(toJson(wrapper)).toMatchSnapshot();
+            expect(wrapper.find('ListItem')).toHaveLength(8);
```

Comment:
why 8?

Reply:
We pass the list of suggestions in that way:
```
suggestionsList = SUGGESTIONS.slice(1, 9);
```
But I agree, it is not obvious. I changed:
```
suggestionsList = SUGGESTIONS.slice(0, 8);
```